.gitignore

```diff
@@ -25,3 +25,7 @@ client/.env
 server/.env
 
 client/.env
+
+client/src/pages/PurchaseCredits/CardForm.js
+client/src/pages/PurchaseCredits/CardSection.js
+client/package-lock.json
```

Comment:
Shouldn't ignore package-lock, not sure why these other files are being ignored either.

Reply:
Yeah all this was temporary. The package-lock had a lot of merge conflicts and i wasn't sure which changes to accept. The other two files may or may not be used when use Material UI with react-stripe-elements, so I figured it was better to not push it and have you guys look at possibly obsolete code.